data/bss-factory-sets.json

```
@@ -0,0 +1,5689 @@
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["electric"],
+        "wantsTera": true,
+        "ability": "swordofruin"
+      }
+    ]
+  },
+  "dragonite": {
+    "weight": 10,
+    "sets": [
+      {
+        "species": "dragonite",
+        "weight": 40,
+        "moves": [
+          ["extremespeed"],
+          ["earthquake"],
+          ["encore", "dragondance"],
+          ["roost"]
+        ],
+        "item": ["rockyhelmet", "leftovers", "heavydutyboots"],
+        "nature": "adamant",
+        "evs": [196, 204, 4, 0, 4, 100],
+        "teratype": ["normal"],
+        "wantsTera": true,
+        "ability": "multiscale"
+      },
+      {
+        "species": "dragonite",
+        "weight": 10,
+        "moves": [["terablast"], ["earthquake"], ["dragondance"], ["roost"]],
+        "item": ["rockyhelmet", "leftovers", "lumberry", "heavydutyboots"],
+        "nature": "adamant",
+        "evs": [196, 204, 4, 0, 4, 100],
+        "teratype": ["flying", "fire", "fairy"],
+        "wantsTera": true,
+        "ability": "multiscale"
+      },
+      {
+        "species": "dragonite",
+        "weight": 5,
+        "moves": [["outrage"], ["extremespeed"], ["earthquake"], ["ironhead"]],
+        "item": ["choiceband"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["normal", "steel"],
+        "wantsTera": true,
+        "ability": "multiscale"
+      },
+      {
+        "species": "dragonite",
+        "weight": 15,
+        "moves": [["outrage"], ["extremespeed"], ["earthquake"], ["terablast"]],
+        "item": ["choiceband"],
+        "nature": "adamant",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["flying"],
+        "wantsTera": true,
+        "ability": "multiscale"
+      },
+      {
+        "species": "dragonite",
+        "weight": 5,
+        "moves": [
+          ["airslash"],
+          ["substitute", "encore"],
+          ["thunderwave"],
+          ["roost"]
+        ],
+        "item": ["rockyhelmet"],
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "bold",
+        "evs": [244, 0, 148, 0, 0, 116],
+        "teratype": ["water", "poison", "fairy"],
+        "wantsTera": true,
+        "ability": "multiscale"
+      },
+      {
+        "species": "dragonite",
+        "weight": 20,
+        "moves": [
+          ["scaleshot"],
+          ["ironhead"],
+          ["earthquake", "lowkick"],
+          ["dragondance"]
+        ],
+        "item": ["loadeddice"],
+        "nature": "jolly",
+        "evs": [4, 252, 0, 0, 0, 252],
+        "teratype": ["steel"],
+        "wantsTera": true,
+        "ability": "multiscale"
+      },
+      {
+        "species": "dragonite",
+        "weight": 5,
+        "moves": [
+          ["firespin"],
+          ["earthquake", "thunderwave"],
+          ["roost"],
+          ["encore"]
+        ],
+        "item": ["leftovers"],
+        "nature": "careful",
+        "evs": [244, 4, 4, 0, 252, 4],
```

```suggestion
        "evs": [252, 4, 0, 0, 252, 0],
```
Lefties numbers aren't a thing.